File: z-clients/javascript/src/models/rateLimitConfigureNamespaceIn.ts
// this file is @generated

export interface RateLimitConfigureNamespaceIn {
    name: string;
}

export const RateLimitConfigureNamespaceInSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): RateLimitConfigureNamespaceIn {
        return {
            name: object['name'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: RateLimitConfigureNamespaceIn): any {
        return {
            'name': self.name,
        };
    }
}

File: z-clients/rust/src/models/rate_limit_configure_namespace_in.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct RateLimitConfigureNamespaceIn {
    pub name: String,
}

impl RateLimitConfigureNamespaceIn {
    pub fn new(name: String) -> Self {
        Self { name }
    }
}
